jwst/coron/align_refs_step.py

```diff
@@ -1,45 +1,54 @@
-""" Replace bad pixels and align psf image with target image."""
+"""Replace bad pixels and align psf image with target image."""
 
 from astropy.nddata.bitmask import interpret_bit_flags
 from stdatamodels.jwst import datamodels
 from stdatamodels.jwst.datamodels.dqflags import pixel
 
 from ..stpipe import Step
 from . import imageregistration
-from . median_replace_img import median_replace_img
+from .median_replace_img import median_replace_img
 
 __all__ = ["AlignRefsStep"]
 
 
 class AlignRefsStep(Step):
-
-    """
-    AlignRefsStep: Align coronagraphic PSF images
-    with science target images.
-    """
+    """Align coronagraphic PSF images with science target images."""
 
     class_alias = "align_refs"
 
     spec = """
         median_box_length = integer(default=3,min=0) # box size for the median filter
         bad_bits = string(default="DO_NOT_USE") # the DQ bit values of bad pixels
-    """ # noqa: E501
+    """  # noqa: E501
 
-    reference_file_types = ['psfmask']
+    reference_file_types = ["psfmask"]
 
     def process(self, target, psf):
-
+        """
+        Execute the AlignRefs calibration step.
+
+        Parameters
+        ----------
+        target : str or DataModel
+            CubeModel or file containing science target data
+        psf : str or DataModel
```

Comment:
Might as well be specific for the input types
```suggestion
        target : str or CubeModel
            CubeModel or file containing science target data
        psf : str or CubeModel
```